### ISSUE TITLE
question: does kvexpress work with consul ACLs, a.k.a. can we specify consul tokens when reading/writing?

### ISSUE THREAD

**Issue by jf** [DataDog/kvexpress#102]
This isn't all that clear to me. The documentation doesn't mention it... and if the open PR #96 is any indication (provided it's still a valid PR and the functionality hasn't gone in elsewhere in some other file), it would appear that the answer is no. If ACLs are in effect, there's no way to pass a token to kvexpress to use?

**Comment by darron** [comment]
Absolutely. If you use the `--token` flag you can set any token you want on the command line.

What that PR did was grab the token from environment variables - which in some situations is worthwhile.
